docs/pages/versions/unversioned/sdk/admob.md

```diff
@@ -14,7 +17,9 @@ Expo includes support for the [Google AdMob SDK](https://www.google.com/admob/)
 
 <InstallSection packageName="expo-ads-admob" />
 
-## Configuration
+## Configuration in app.json / app.config.js
+
+You can configure `expo-ads-admob` using its built-in [config plugin](../../../guides/config-plugins.md) if you use config plugins in your project ([EAS Build](../../../build/introduction.md) or `expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
 
```

Comment:
i think we should also have a section here addressing people using classic builds, indicating how to configure the tracking permissions message

Reply:
Sure, I can add it, but I thought the `IOSPermissions` block was for that?

Reply:
added